contracts/ClaimsManager.sol

```diff
@@ -269,6 +269,48 @@ contract ClaimsManager is
         );
     }
 
+    function updatePolicy(
```

Comment:
This function assumes that params are valid because they come from a signed offer created by us, right? I'm just wondering why not call `createPolicy` or `upgradePolicy` functions to perform all the `require` statements in those functions. I'm guessing that it is to avoid calculating the `policyHash` twice but in that case we could keep it as is and move the require statements to separate functions so they can then be called from `updatePolicy` and from `createPolicy`/`upgradePolicy`. We could also benefit from not having to check `msg.sender` is admin and rely on the events emitted from those functions.

Reply:
Also maybe a better name would be `upsertPolicy` 😁

Reply:
Interesting, I'll try it out